mne/io/fieldtrip/tests/test_fieldtrip.py

```diff
@@ -228,3 +228,41 @@ def test_one_channel_elec_bug(version):
 
     with pytest.warns(**no_info_warning):
         mne.io.read_raw_fieldtrip(fname, info=None)
+
+
+@testing.requires_testing_data
+# Reading the sample CNT data results in a RuntimeWarning because it cannot
+# parse the measurement date. We need to ignore that warning.
+@pytest.mark.filterwarnings('ignore::RuntimeWarning')
```

Comment:
Could you add a suitable regex in the `::` so that it's more targeted? That way other warnings that we should pay attention to won't creep in easily.